ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/platform.cfg

```diff
@@ -5,6 +5,29 @@ transceiver qsfp default-mode 4x10G
 hostname platform
 !
 platform trident forwarding-table partition 2
+platform trident mmu queue profile mc_example_profile apply
```

Comment:
when I apply this on a dut running 4.30, this command comes at the very end of the config.

Reply:
Agreed, changed the template accordingly.